src/Converter.php

```diff
@@ -110,11 +122,23 @@ public function toDateTimeLocalString($unitPrecision = 'second')
     /**
      * Format the instance with day, date and time
      *
+     * @param string $unitPrecision
+     *
+     * @return string
+     */
+    public function toDayDateTimeString($unitPrecision = 'second')
+    {
+        return $this->format('l j F Y ' . static::getTimeFormatByPrecision($unitPrecision));
+    }
+
+    /**
+     * Format the instance with the year, and a readable month
+     *
      * @return string
      */
-    public function toDayDateTimeString()
+    public function toFormattedMonthYearString(): string
     {
-        return $this->format('l j F Y g:i A');
+        return $this->format('F Y');
     }
```

Comment:
Why you removed this?

Reply:
I have not removed it.
In the Diff viewer, it looks like it has been removed.
I changed `toDayDateTimeString` function from
```php
public function toDayDateTimeString()
{
    return $this->format('l j F Y g:i A');
}
```
To
```php
public function toDayDateTimeString($unitPrecision = 'second')
{
    return $this->format('l j F Y ' . static::getTimeFormatByPrecision($unitPrecision));
}
```
And add new method `toFormattedMonthYearString`
```php
public function toFormattedMonthYearString()
{
    return $this->format('F Y');
}
```